app/models/task.py

```diff
@@ -3,3 +3,47 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
```

Comment:
Fun fact: the default value for nullable is `True` so we don't need to explicitly add it!